packages/client/internals/PresenterPrint.vue

```diff
@@ -47,5 +47,19 @@ const slidesWithNote = computed(() => rawRoutes
 <style lang="postcss">
 @page {
   size: A4;
+  margin-top: 1.5cm;
+  margin-bottom: 1cm;
+}
```

Comment:
Add default print margins, prevent to have text too near the top and the bottom.